library/Mockery/Expectation.php

```diff
@@ -558,6 +558,28 @@ public function andReturnUsing(...$args)
         return $this;
     }
 
+    /**
+     * Sets up a closure to return the nth argument from the expected method call
+     *
+     * @param int $index
+     * @return self
+     */
+    public function andReturnArgument($index)
```

Comment:
I'm not 100% sure what would be best, but in other public methods, we use `withArgs`, rather than `withArguments`. 

So would we better with `andReturnArg` or is `Arg` too ambiguous?

Reply:
I think `andReturnArg` is fine; I'm happy to keep things consistent.